Packs/MajorBreachesInvestigationandResponse/ReleaseNotes/1_6_31.md

```diff
@@ -0,0 +1,71 @@
+
+Note: This is a beta playbook, which lets you implement and test pre-release software. Since the playbook is beta, it might contain bugs. Updates to the pack during the beta phase might include non-backward compatible features. We appreciate your feedback on the quality and usability of the pack to help us identify issues, fix them, and continually improve.
+- Modified `fromversion: 6.0.0` and `toversion: 6.4.9` 
+- Added new configuration to support `fromversion:6.5.0` and upwards
+
+##### HAFNIUM - Exchange 0-day exploits
+
+- Replaced the **Block Indicators - Generic v2** sub-playbook with the new **Block Indicators - Generic v3** playbook. (Available from Cortex XSOAR 6.5.0).
+- Added input `AutoBlockIndicators` - This input determines if to block the indicators automatically or manually using the sub-playbook `Block Indicators - Generic v3`. (Available from Cortex XSOAR 6.5.0).
+- Added input `UserVerification` - This input determines if blocking IPs is should be done with or without user verification.  The IPs blocking is done by using the sub-playbook `Block Indicators - Generic v3`. (Available from Cortex XSOAR 6.5.0). 
+- Modified `fromversion: 6.0.0` and `toversion: 6.4.9` 
+- Added new configuration to support `fromversion:6.5.0` and upwards
+
+##### SolarStorm and SUNBURST Hunting and Response Playbook
+
+- Replaced the **Block Indicators - Generic v2** sub-playbook with the new **Block Indicators - Generic v3** playbook. (Available from Cortex XSOAR 6.5.0).
+- Added input `AutoBlockIndicators` - This input determines if to block the indicators automatically or manually using the sub-playbook `Block Indicators - Generic v3`. (Available from Cortex XSOAR 6.5.0).
+- Added input `UserVerification` - This input determines if blocking IPs is should be done with or without user verification.  The IPs blocking is done by using the sub-playbook `Block Indicators - Generic v3`. (Available from Cortex XSOAR 6.5.0).
+- Removed expanse-get-risky-flows and expanse-list-risk-rules as they are EOD with no suitable replacements
+- Modified `fromversion: 6.0.0` and `toversion: 6.4.9` 
+- Added new configuration to support `fromversion:6.5.0` and upwards
+- 
+##### NOBELIUM - wide scale APT29 spear-phishing
+
+- Replaced the **Block Indicators - Generic v2** sub-playbook with the new **Block Indicators - Generic v3** playbook. (Available from Cortex XSOAR 6.5.0).
+- Added input `AutoBlockIndicators` - This input determines if to block the indicators automatically or manually using the sub-playbook `Block Indicators - Generic v3`. (Available from Cortex XSOAR 6.5.0).
+- Added input `UserVerification` - This input determines if blocking IPs is should be done with or without user verification.  The IPs blocking is done by using the sub-playbook `Block Indicators - Generic v3`. (Available from Cortex XSOAR 6.5.0).
+- Modified `fromversion: 6.0.0` and `toversion: 6.4.9` 
```

Comment:
The following lines should be removed:

Added configuration fromversion: 5.5.0 and toversion: 6.4.9
Added configuration fromversion: 6.5.